apps/journeys-admin/src/components/ReportButtons/ReportButtons.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import { Story, Meta } from '@storybook/react'
+import { MockedProvider } from '@apollo/client/testing'
+import { journeysAdminConfig } from '../../libs/storybook'
+import { ReportButtons } from "./ReportButtons"
+
+const ReportButtonsStory = {
+  ...journeysAdminConfig,
```

Comment:
```suggestion
  ...simpleComponentConfig,
```

This just cuts down on the snapshots we take since this component won't change depending on screen size